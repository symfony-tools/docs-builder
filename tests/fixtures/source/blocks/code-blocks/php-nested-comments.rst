You can do that by adding a "stamp" to your message::

    use Symfony\Component\Messenger\MessageBusInterface;
    use Symfony\Component\Messenger\Stamp\DelayStamp;

    public function index(MessageBusInterface $bus)
    {
        $bus->dispatch(new SmsNotification('...'), [
            // wait 5 seconds before processing
            new DelayStamp(5000),
        ]);
    }
