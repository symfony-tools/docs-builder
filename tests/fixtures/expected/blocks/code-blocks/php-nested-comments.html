<p>You can do that by adding a "stamp" to your message:</p>
<div translate="no" data-loc="10" class="notranslate codeblock codeblock-length-md codeblock-php">
    <div class="codeblock-scroll">
        <pre class="codeblock-lines">1
2
3
4
5
6
7
8
9
10</pre>
        <pre class="codeblock-code">
            <code>
                <span class="hljs-keyword">use</span>
                <span class="hljs-title">Symfony</span>
                \
                <span class="hljs-title">Component</span>
                \
                <span class="hljs-title">Messenger</span>
                \
                <span class="hljs-title">MessageBusInterface</span>
                ;
                <span class="hljs-keyword">use</span>
                <span class="hljs-title">Symfony</span>
                \
                <span class="hljs-title">Component</span>
                \
                <span class="hljs-title">Messenger</span>
                \
                <span class="hljs-title">Stamp</span>
                \
                <span class="hljs-title">DelayStamp</span>
                ;
                <span class="hljs-keyword">public</span>
                <span class="hljs-function">
                    <span class="hljs-keyword">function</span>
                    <span class="hljs-title">index</span>
                    <span class="hljs-params">
                        (MessageBusInterface
                        <span class="hljs-variable">
                            <span class="hljs-variable-other-marker">$</span>
                            bus
                        </span>
                        )
                    </span>
                </span>
                {
                <span class="hljs-variable">
                    <span class="hljs-variable-other-marker">$</span>
                    bus
                </span>
                -&gt;
                <span class="hljs-title invoke__">dispatch</span>
                (
                <span class="hljs-keyword">new</span>
                <span class="hljs-title invoke__">SmsNotification</span>
                (
                <span class="hljs-string">'...'</span>
                ), [
                <span class="hljs-comment">// wait 5 seconds before processing</span>
                <span class="hljs-keyword">new</span>
                <span class="hljs-title invoke__">DelayStamp</span>
                (
                <span class="hljs-number">5000</span>
                ), ]);
}
            </code>
        </pre>
    </div>
</div>
